remote-messaging/remote-messaging-impl/src/main/java/com/duckduckgo/remote/messaging/impl/RemoteMessagingConfigMatcher.kt

```diff
@@ -29,12 +27,9 @@ import com.duckduckgo.remote.messaging.impl.models.MatchingAttribute.Unknown
 import timber.log.Timber
 
 class RemoteMessagingConfigMatcher(
-    @DeviceAttrMatcher val deviceAttributeMatcher: AttributeMatcher,
-    @DeviceAttrMatcher val androidAppAttributeMatcher: AttributeMatcher,
```

Comment:
This was probably a but, it should've been annotated with `AndroidAttrMatcher` instead.